flake.lock

```diff
@@ -461,6 +462,23 @@
         "type": "github"
       }
     },
+    "stevenblack-hosts": {
```

Comment:
steven who?

Reply:
https://github.com/StevenBlack/hosts a source for dumb adblockers